composer.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Yea, Basically, we have to put `composer.json` at top level of repository.

(little bit off topic) I've tested sub directory version which using github svn support.
it works good on local machine. but packagist detects the url as git :( 

e.g) https://github.com/chobie/composer-test


But now, current PHP implementation is still experimental. we can postpone this decision till stable release.